app.py

```diff
@@ -0,0 +1,53 @@
+from flask import Flask, jsonify, request, make_response
+from datetime import datetime
+
+app = Flask(__name__)
+
+entries = [
+    {
+        'entryId':1,
+        'title':'dictionary function',
+        'content':'using comprehenion',
+        'time':'2018-09-14 11:31:09.318212'
+        
+    },
+    {
+        'entryId':2,
+        'title':'bankaccount class',
+        'content':'class implementation corresponding to the provided tests',
+        'time':'2018-09-21 10:1:09.318212'
+    }
+        ]
+
+def get_timestamp():
+    return datetime.now().strftime(("%Y-%m-%d %H:%M:%S"))
+
+@app.route('/api/v1/entries', methods=['GET'])
```

Comment:
If you are using get only. You don't need the `methods=['GET']` drama

Reply:
Alright, will change this